pkg/totalaccountwatcher/totalaccountwatcher.go

```diff
@@ -40,7 +44,7 @@ func Initialize(client client.Client, watchInterval time.Duration) {
 	// Builder in their struct and uses it to GetClient() dynamically as needed. This one grabs a
 	// single client one time and stores it in a global.
 	builder := &awsclient.RealBuilder{}
```

Comment:
#430 just merged, which renamed this to `Builder`, so you might need to rebase. But maybe not, if github is smart enough :)

Reply:
Github is smart, but if I want to pull locally and review, then to make sure everything is working as expeted, it would need a rebase off master

Reply:
Looks like it wants an explicit rebase anyway. (Hm, it has the red X from tide, but not the needs-rebase label.)